### PR TITLE
bugfix/3038/search-button-alignment

### PR DESCRIPTION
**Pull request by Tarun1505**
Target #3038 

### Description
   Fixed Search button alignment on working staff page .

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
         "cypress-open": "node_modules/.bin/cypress open",
         "cypress-record": "node_modules/.bin/cypress run --record --key",
         "dev": "npm run development",
-        "development": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
-        "watch": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+        "development": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress  --config=node_modules/laravel-mix/setup/webpack.config.js",
```
Comment (Ayush8923) [inline package.json:8]: These changes are not needed. Please revert this file changes.

File: package-lock.json [hunk 2]
```diff
@@ -6926,7 +6926,72 @@
             "resolved": "https://registry.npmjs.org/fsevents/-/fsevents-1.2.12.tgz",
             "integrity": "sha512-Ggd/Ktt7E7I8pxZRbGIs7vwqAPscSESMrCSkx2FtWeqmheJgCo2R74fTsZFCifr0VTPwqRpPv17+6b8Zp7th0Q==",
             "bundleDependencies": [
-                "node-pre-gyp"
+                "node-pre-gyp",
```
Comment (Ayush8923) [inline package-lock.json:6929]: Same here. This file changes are not needed. Please revert these file changes.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Left some comments. Please check.

File: package.json [hunk 3]
```diff
@@ -52,4 +52,4 @@
         "resources/js/*.(js|vue)": "./node_modules/.bin/eslint --ext .js,.vue --fix",
         "Modules/*/resources/assets/js/*.(js|vue)": "./node_modules/.bin/eslint --ext .js,.vue --fix"
     }
-}
+}
```
Comment (Ayush8923) [inline package.json:55]: Please revert this file. It's not needed.

File: resources/views/hr/employees/index.blade.php [hunk 4]
```diff
@@ -15,9 +15,9 @@
                     <option {{ $filters['status'] == 'previous' ? "selected=selected" : '' }} value="previous">Previous</option>
                 </select>
             </div>
-            <div class="d-flex align-items-center ml-35">
-                    <input type="text" name="employeename" class="form-control" id="name" placeholder="Enter the Employee" value="{{ request()->get('employeename') }}">
-                    <button class="btn btn-info ml-2 text-white">Search</button>
+            <div class="d-flex align-items-center ml-25">
+                    <input type="text" name="employeename" class="form-control" id="name" placeholder="Enter the Employee" value="{{ request()->get('employeename') }}" button class="pr-3.0">
```
Comment (Ayush8923) [inline resources/views/hr/employees/index.blade.php:19]: Please let me know, how much size of the `pr-3.0`. and also the what is means of the `button` in this input field.
```suggestion
                    <input type="text" name="employeename" class="form-control pr-3.0" id="name" placeholder="Enter the Employee" value="{{ request()->get('employeename') }}" button class="">
```

File: resources/views/hr/employees/index.blade.php [hunk 5]
```diff
@@ -15,9 +15,9 @@
...
+                    <button class="btn btn-info ml-1 pr-1 py-1.0 text-white">Search</button>
```
Comment (Ayush8923) [inline resources/views/hr/employees/index.blade.php:20]: Please let me know, how much size of the `pr-1.0`.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Left some comments. And please revert the `package-lock.json` and `package.json` file from this PR. It's not needed in this PR.

File: resources/views/hr/employees/index.blade.php (hunk 5 above)
Comment (Tarun1505) [reply resources/views/hr/employees/index.blade.php:20]: It has a size of 1px .

File: resources/views/hr/employees/index.blade.php (hunk 4 above)
Comment (Tarun1505) [reply resources/views/hr/employees/index.blade.php:19]: This **button**  is added by mistake , and I have corrected it .

**Comment by Ayush8923** [comment]
- You can see in this below screenshot, the alignment of the search bar is not correct. The alignment of the search bar should be at the end of the table. and also you need to check in all the screen sizes.

Screenshot 2023-05-13 at 9 12 50 PM

**Comment by Tarun1505** [comment]
> [May 2, 2023 16:18 ](#event-9144323225)

Can you please elaborate it for better understanding!

**Comment by GauravGusain98** [comment]
Due to inactivity closing this PR

**Pull request closed without merging**